compiler/optimizer/LoopReplicator.cpp

```diff
@@ -147,6 +147,8 @@ int32_t TR_LoopReplicator::perform(TR_Structure *str)
 
    if (!region->isNaturalLoop())
       {
+      TR::DebugCounter::incStaticDebugCounter(comp(), TR::DebugCounter::debugCounterName(comp(), "LoopReplicator/NotNaturalLoop/%s/(%s)/region_%d",
+         comp()->getHotnessName(comp()->getMethodHotness()), comp()->signature(), region->getNumber()));
```

Comment:
This case is unlikely ever to be interesting, unless perhaps `region` is an improper region. For example, I think it would always count at least `LoopReplicator/NotNaturalLoop/*/(*)/region_0`

Reply:
Is it possible to use shorter lines? It's hard to read the diffs because they scroll horizontally